# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/ipc/PRemoteDecoderManager.ipdl
# Commit: 3f08ccb1f141
# Full Hash: 3f08ccb1f141069bfd90474d6023413303307ec4
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2020-11-04 09:55:18
# Regressor Bug: 1595994
# File Overlap Count: 1
# Description:
#   Bug 1674043 - P6. Remove sync PDecoderMananger::Supports API. r=mattwoodrow,bryce,mjf,ipc-reviewers,nika
#   
#   In bug 1595994 we attempted to streamline the ability to determine which decoder was available regardless of the process they would be running in. This was subsequently done via the PDMFactory.
#   
#   As there are several JS API that can query which codec are supported, it requires a synchronous mechanism.
# ==============================================================================

diff -r 33563a16a5c3 -r 3f08ccb1f141 dom/media/ipc/PRemoteDecoderManager.ipdl
--- a/dom/media/ipc/PRemoteDecoderManager.ipdl	Mon Nov 02 15:32:59 2020 +0000
+++ b/dom/media/ipc/PRemoteDecoderManager.ipdl	Wed Nov 04 02:22:33 2020 +0000
@@ -33,11 +33,6 @@
   manages PRemoteDecoder;
 
 parent:
-  sync Supports(RemoteDecoderInfoIPDL info,
-                TextureFactoryIdentifier? identifier)
-         returns (bool success,
-                  DecoderDoctorDiagnostics diagnostics);
-
   sync PRemoteDecoder(RemoteDecoderInfoIPDL info,
                       OptionSet options,
                       TextureFactoryIdentifier? identifier)